Frontend/ui-driver/src/Screens/HomeScreen/ComponentConfig.purs

```diff
@@ -367,7 +368,9 @@ paymentPendingPopupConfig state =
     backgroundClickable = true,
     cornerRadius = (PTD.Corners 15.0 true true true true),
     coverImageConfig {
-      imageUrl = "ny_ic_payment_pending," <> getAssetStoreLink FunctionCall <> "ny_ic_payment_pending.png"
+      imageUrl = case popupType of
+                          GO_ONLINE_BLOCKER  -> "ny_ic_payment_pending," <> getAssetStoreLink FunctionCall <> "ny_ic_payment_pending.png"
+                          _ ->  "ny_ic_clear_dues_early," <> getAssetStoreLink FunctionCall <> "ny_ic_payment_pending.png"
```

Comment:
@prakharritik is this asset added to asset store.

Reply:
Yes

Reply:
@prakharritik the image name from native and remote doesn't match here.